parea/schemas/models.py

```diff
@@ -39,8 +51,8 @@ class LLMInputs:
     model: Optional[str] = None
     provider: Optional[str] = None
     model_params: Optional[ModelParams] = None
-    messages: Optional[List[Message]] = None
-    functions: Optional[List[Any]] = None
+    messages: Optional[list[Message]] = None
```

Comment:
I think that annotation breaks some old python versions

Reply:
Which "`list`" ?  vs `List`?

Reply:
ah interesting.